client/src/pages/Programmes/scrollHook.js

```diff
@@ -0,0 +1,20 @@
+import { useRef, useEffect } from "react";
+
+export const useHorizontalScroll = () => {
+  const elRef = useRef();
+  useEffect(() => {

```

Comment:
![image](https://user-images.githubusercontent.com/39323194/95695211-625db900-0c92-11eb-9d59-34e11f38369a.png)
Are you able to make this part scroll

Reply:
That part scrolls for me on my pc

Reply:
hmm, it only fails in editing mode but the other ones work

Reply:
It fails for me too, even when I scroll horizontally with my trackpad

Reply:
Fixed ref reference, can you guys try again